swtbot_test/src/iotLTStestsuites/CodeGeneration.java

```diff
@@ -76,7 +76,7 @@ public void tc_00_ChangeRTOSLocation() throws Exception{
 	
 	@Test
 	public void tc_01_CreateIoTLTSProject() throws Exception {
-		PGUtility.createProject(RTOSType.FREERTOSIOTLTS, RTOSVersion.IoTLTS_202210_1_0_0, RTOSApplication.IOT_LTS_ETHER_PUBSUB,
+		PGUtility.createProject(RTOSType.FREERTOSIOTLTS, RTOSVersion.IoTLTS_202210_1_0_0, RTOSApplication.KERNEL_BARE,
```

Comment:
Q: This change based on new package structure?

Reply:
As I checked, In the freertos.xml of package LTS 1.1.3, the Application is still "pubsub_ether", in my new latest patchset I have fix this